src/smspark/bootstrapper.py

```diff
@@ -36,11 +37,14 @@ class Bootstrapper:
     HADOOP_CONFIG_PATH = "/opt/hadoop-config/"
     HADOOP_PATH = "/usr/lib/hadoop"
     SPARK_PATH = "/usr/lib/spark"
+
     HIVE_PATH = "/usr/lib/hive"
     PROCESSING_CONF_INPUT_PATH = "/opt/ml/processing/input/conf/configuration.json"
     PROCESSING_JOB_CONFIG_PATH = "/opt/ml/config/processingjobconfig.json"
     INSTANCE_TYPE_INFO_PATH = "/opt/aws-config/ec2-instance-type-info.json"
     EMR_CONFIGURE_APPS_URL = "https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-configure-apps.html"
+    JAR_DEST = SPARK_PATH + "/jars"
+    OPTIONAL_JARS = {"jets3t-0.9.0.jar": HADOOP_PATH + "/lib"}
```

Comment:
Could you leave a comment explaining:

* why we optionally copy this JAR
* in which release versions we expect this JAR to be copied
* what this JAR is needed for in the first place

Reply:
- why we optionally copy this JAR - it's needed by spark 2.4 s3a client
- in which release versions we expect this JAR to be copied - only 2.4, we don't need it for 3.0
- what this JAR is needed for in the first place - we are going to get some not found exception without this jar for spark 2.4